tests/clip_vertex_data_test.py

```diff
@@ -136,10 +136,15 @@ def test_clip_groups(tmp_path):
                 },
             }
         )
+
         with Workspace(tmp_path / r"testClipPoints_copy.geoh5") as new_workspace:
-            group_a.copy_from_extent(extent, parent=new_workspace)
+            group_a.copy_from_extent(extent, parent=new_workspace, clear_cache=True)
```

Comment:
**suggestion (tests):** have a test test with `clear_cache=False`

and also tests for `Workspace.copy_to_parent()` and  implementation `Entity.copy()` with `clear_cache=True` and `clear_cache=False`